packages/ethereum-storage/src/ethereum-utils.ts

```diff
@@ -1,7 +1,7 @@
 import { StorageTypes } from '@requestnetwork/types';
 import * as config from './config';
 
-const bigNumber: any = require('bn.js');
+import * as BigNumber from 'bn.js';
```

Comment:
the `require` was a pain. I love that we can get rid of it :slightly_smiling_face: